### PR TITLE
[pixman] Remove collect_libs and use library real name

### PR DESCRIPTION
**Pull request by uilianries**
Specify library name and version:  **pixman/0.40.0**

The Pixman library is missing in ConanCenter for Debug build type. I tried to generate again via Tapaholes, but is still missing. This PR should generate a new recipe revision and new packages, fixing the problem (I hope). 

Related to https://github.com/conan-io/conan-center-index/pull/17844

/cc @ericLemanissier 

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by ericLemanissier — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

File: recipes/pixman/all/conandata.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 sources:
   "0.40.0":
-    url: "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
+    url:
+    - "https://gitlab.freedesktop.org/pixman/pixman/-/archive/pixman-0.40.0/pixman-pixman-0.40.0.tar.gz"
+    - "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
```
Comment (ericLemanissier) [inline recipes/pixman/all/conandata.yml:5]:
```suggestion
    url: "https://www.cairographics.org/releases/pixman-0.40.0.tar.gz"
```
the gitlab archive does not have the same sha256, it is automatically generated by gitlab, contrary to the release tarball on cairographics.org

Comment (uilianries) [reply recipes/pixman/all/conandata.yml:5]: I thought I didn't commit that change, I saw it locally too, but it seems like commit -a was faster than me. Thank you for catching it, I'll remove it.

Comment (uilianries) [reply recipes/pixman/all/conandata.yml]: Done.

**Review by ericLemanissier — APPROVED** [review]

**Comment by uilianries** [comment]
We need a mirror for it, another download failure 😞

**Pull request merged**